From 5c03a7931f5f14e952bce9a06e0fab634282072e Mon Sep 17 00:00:00 2001
From: Pramod Kumar <pramod.kumar_1@nxp.com>
Date: Fri, 14 Sep 2018 16:35:48 +0530
Subject: [PATCH 106/107] ls1088a: remove dhcp function from u-boot env as boot source

dhcp as boot_targets is attempted first before qspi boot
which causes the boot delay.

Signed-off-by: Pramod Kumar <pramod.kumar_1@nxp.com>
---
 include/configs/ls1088ardb.h |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/include/configs/ls1088ardb.h b/include/configs/ls1088ardb.h
index 40b1716..5752696 100644
--- a/include/configs/ls1088ardb.h
+++ b/include/configs/ls1088ardb.h
@@ -464,8 +464,7 @@
 
 #define BOOT_TARGET_DEVICES(func) \
 	func(MMC, mmc, 0) \
-	func(SCSI, scsi, 0) \
-	func(DHCP, dhcp, na)
+	func(SCSI, scsi, 0)
 #include <config_distro_bootcmd.h>
 #endif
 
-- 
1.7.1

